server/README.md

```diff
@@ -3,8 +3,29 @@
 ## About
 This folder contains all the necessary files to build and run a simple model aggregator in an enclave environment. The aggregator takes the element-wise sum of the variables received from multiple clients, weighs them by local iterations, then adds them to the existing model. All sensitive computation is done in an enclave environment, with encryption/decryption being performed inside.
 
+## Dependencies
+Building the server code requires the following dependencies that can be installed on Ubuntu as follows:
+1. Pip3:
+    * `sudo apt install python3-pip`
+2. CMake:
+    * `pip3 install cmake`
+1. FlatBuffers:
+    * `sudo apt install snapd`
+    * `sudo snap install flatbuffers`
```

Comment:
These instructions do not work on Docker containers, and the way I was able to get around it involves changing the path to the flatbuffers install in `CMakeLists.txt`, so it does not work with the current, existing code. Perhaps one of you would like to amend this?